.editorconfig

```diff
@@ -0,0 +1,33 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.py]
+indent_size = 4
+
+[{.git*,Makefile}]
+indent_style = tab
+
+[{COMMIT_EDITMSG,MERGE_MSG,SQUASH_MSG}]
+max_line_length = 72
+
+[**.md]
+# double whitespace at end of line
+# denotes a line break in Markdown
+trim_trailing_whitespace = false
+x-soft-wrap-text = true
+
+[*.json]
+insert_final_newline = ignore
+
+[*{.ini,.flake8}]
+indent_size = 4
```

Comment:
Wondering why this isn't 2? (real question...I'm not sure the diff on this vs above)

Reply:
Only reason I chose 4 here was that was what was used in the existing files in the repo. I can remove it, and ini/flake8 files will inherit from [*] above (indent_size = 2).